CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,40 @@
+# Fuel Rats Contributor Code of Conduct
+
+Please read through this document and understand the specifics.  If you do not, feel free to ask another contributor, or contact a member of the Moderation team.
+
+This policy is in place to establish behavioral expectations and guidelines to ensure an open, tolerant, and professional environment for all participants.  We believe anyone participating in our projects deserve a harassment-free experience, regardless of age, caste, gender, gender identity or expression, culture ethnicity, disability, language, political beliefs, profession, race, religion, sexual orientation, socioeconomic status, or technical ability.  Discrimination based on any of these protected characteristics will not be tolerated.
+
+This policy is NOT intended to supersede the Fuel Rats [Terms of Service](https://fuelrats.com/terms-of-service), but as a supplement to cover non-rescue contributions.
+
+## Standards
+
+We hope all members strive to go beyond examples laid out here and actively create a positive and respectful atmosphere.
+
+* **Foster a friendly, judgement-free environment for all.**  This project houses international contributors from all walks of life, with varying experience.
+* **Keep an open-mind, and respect differing viewpoints.** Lest we forget the professional attitude best displayed in our work.
+* **Be professional and considerate.** Discuss and resolve conflicts in an open, civil manner.  Disagreements are inevitable.
+* **Show patience and be helpful.**  We all make mistakes.  Playing the 'blame game' helps no one.
+
+#### Negative Behavior Examples
+
+* Forcing others to participate
+* Public or private harassment
+* Trolling/Personal attacks
+* 'Doxxing' or making a person's private information available without explicit permission
+* Conduct considered inappropriate for a professional environment.
+* Conduct considered unbecoming by any member of the 'Tech Rat Moderation' team.
```

Comment:
I agree we need to be more specific here. Some examples would be great, and if not what we have otherwise will have to be sufficient.

Reply:
> A CoC should establish a spirit of what we want, and make it clear that if someone chooses to act outside that framing, they may get smacked for it.

I think that `Conduct considered unbecoming by any member of the 'Tech Rat Moderation' team.` doesn't even come close to make that framing clear, and just makes it harder to judge XD

Reply:
Well, we can have it one way, or the other.  Either we list out a very comprehensive set of disallowed behaviors, or we make a general statement regarding things that go against the spirit of the agreement.  The intent with the current wording is to provide examples, but also leave room for interpretation should something come up that isn't explicitly covered in the examples.

Reply:
I modified the wording a bit, hoping to make this more general.  ``Keep an Open Mind`` has been removed, in favor of simply ``Respect differing viewpoints``.  If the suggestion that comes across is that you must hear out every banal idea that gets thrown out, that isn't the intent.

Reply:
I'm still iffy on

> Conduct considered inappropriate for a professional environment.

myself... again back to the highly up to personal interpretation thing.

Reply:
I think that line is too broad, 
Like Clapton said its open to personal interpretation. 

The problem with personal interpretation is it becomes all to easy to claim "well it seems perfectly acceptable to me".

I think we need to define with a bit more specifics what is considered professional.

Reply:
The opposite to that is of course "Well it's not in the CoC"... inevitably this means we will have to expand beyond what is written, either because it is broad or we encounter a situation that is not specified.